cmd/influx/pkg.go

```diff
@@ -345,6 +345,74 @@ func (b *cmdPkgBuilder) cmdPkgValidate() *cobra.Command {
 	return cmd
 }
 
+func (b *cmdPkgBuilder) cmdStack() *cobra.Command {
+	cmd := b.newCmd("stack", nil, false)
+	cmd.Short = "Stack command is the container for all stack CRUD"
```

Comment:
Could just be `"stack management commands"` to be consistent with other descriptions.